File: Beecrowd/2031 Pedra, Papel, Ataque Aéreo/main.cpp
#include <iostream>

using namespace std;

int main(){
    int n;
    string jog1, jog2;

    cin >> n;

    while(n--){
        cin >> jog1;
        cin >> jog2;
        
        if((jog1 == "ataque") and (jog2 == "pedra") or (jog1 == "pedra") and (jog2 == "papel") or (jog1 == "ataque") and (jog2 == "papel"))
            cout << "Jogador 1 venceu" << endl;
        else if((jog2 == "ataque") and (jog1 == "pedra") or (jog2 == "pedra") and (jog1 == "papel") or (jog1 == "papel") and (jog2 == "ataque"))
            cout << "Jogador 2 venceu" << endl;
        else if((jog1 == "papel") and (jog2 == "papel"))
            cout << "Ambos venceram" << endl;
        else if((jog1 == "pedra") and (jog2 == "pedra"))
            cout << "Sem ganhador" << endl;
        else
            cout << "Aniquilacao mutua" << endl;
    }

    return 0;
}

File: Beecrowd/2031 Pedra, Papel, Ataque Aéreo/main.py
n = int(input())

for _ in range(n):
    jog1 = input()
    jog2 = input()
    
    if (jog1 == "ataque") and (jog2 == "pedra") or (jog1 == "pedra") and (jog2 == "papel") or (jog1 == "ataque") and (jog2 == "papel"):
        print("Jogador 1 venceu")
    elif (jog2 == "ataque") and (jog1 == "pedra") or (jog2 == "pedra") and (jog1 == "papel") or (jog1 == "papel") and (jog2 == "ataque"):
        print("Jogador 2 venceu")
    elif (jog1 == "papel") and (jog2 == "papel"):
        print("Ambos venceram")
    elif (jog1 == "pedra") and (jog2 == "pedra"):
        print("Sem ganhador")
    else:
        print("Aniquilacao mutua")